### PR TITLE
Create a stub of a file for Orland Park, IL

### PR DESCRIPTION
**Pull request by neumsTraining2016**
This is my first pull request

### CODE REVIEW

File: places/neumsTraining2016-OrlandPark.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Orland Park, IL
+
+### Places to Eat 
+Every franchise/chain restaurant you can imagine
+Traversos
+Chesdans
+Aurelios
```
Comment (laurahayden) [inline places/neumsTraining2016-OrlandPark.md]: I like Aurelios too.

File: places/neumsTraining2016-OrlandPark.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## Orland Park, IL
```
Comment (pindel) [inline places/neumsTraining2016-OrlandPark.md]: where is Orland Park?

File: places/neumsTraining2016-OrlandPark.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+### Places to Eat 
+Every franchise/chain restaurant you can imagine
```
Comment (pindel) [inline places/neumsTraining2016-OrlandPark.md]: Is there a McDoanlds?

File: places/neumsTraining2016-OrlandPark.md (hunk 2 above)
Comment (neumsTraining2016) [inline places/neumsTraining2016-OrlandPark.md]: 50 miles directly south of Riverwoods.  131st to 159th LaGrange Road, Harlem (R43)

**Comment by pindel** [comment]
Great moves neumsTraining2016, keep it up!

**Pull request merged**